packages/ui/lib/Tag/index.js

```diff
@@ -1,27 +1,30 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
+import { p_12_medium } from "../styles/textStyles";
+
+const colors = {
+  active: "#6848FF",
+  resolved: "#E2E8F0",
+};
 
 const Wrapper = styled.span`
   padding: 2px 12px;
-  text-transform: capitalize;
   color: white;
-  line-height: 18px;
+  line-height: 16px;
+  font-size: 12px;
+  ${p_12_medium}
+  background-color: ${(p) => colors[p.status] || p.color || "#000"};
```

Comment:
imo, would it be better to  put `p.color` before `colors[p.status]` ?

Reply:
the color prop only for custom content

❌ and it is not design for both: `<Tag status="active" color="orange">my content</Tag>`